### ISSUE TITLE
Show non-demangled class name in OBJC JSON output

### ISSUE THREAD

**Issue by radare** [radare/radare2#5566]

**Comment by ret2libc** [comment]
Is this still relevant @trufae ?

**Comment by trufae** [comment]
yep it is, the output of icj only shows the class/method/field name demangled. there's no flagname or non-demangled-name fields

**Comment by trufae** [comment]
This also applies to c++, dex, java, .. the OBJC in the title can be misleading
